test/Microsoft.DotNet.ProjectModel.Tests/GivenThatIWantToLoadAProjectJsonFile.cs

```diff
@@ -747,7 +747,7 @@ public void It_adds_the_dependency_when_the_version_is_set_directly()
             dependency.Type.Should().Be(LibraryDependencyType.Default);
             dependency.SourceFilePath.Should().Be(ProjectFilePath);
             dependency.SourceLine.Should().Be(3);
-            dependency.SourceColumn.Should().Be(31);
+            dependency.SourceColumn.Should().Be(30);
         }
```

Comment:
@javiercn why did these values change in your PR? I followed yours for the fixes to make this test work. What led you to change these values?